music21/note.py

```diff
@@ -1776,6 +1776,7 @@ class Unpitched(NotRest):
 
     def __init__(self, displayName=None, **keywords):
         super().__init__(**keywords)
+        self._chordAttached: t.Optional['music21.percussion.PercussionChord'] = None
```

Comment:
This was already being set in `ChordBase._add_core_or_init()`, but without being initialized anywhere.